kimchi/src/verifier_index.rs

```diff
@@ -76,7 +76,7 @@ pub struct VerifierIndex<G: KimchiCurve> {
     pub sigma_comm: [PolyComm<G>; PERMUTS],
     /// coefficient commitment array
     #[serde(bound = "PolyComm<G>: Serialize + DeserializeOwned")]
-    pub coefficients_comm: [PolyComm<G>; COLUMNS],
+    pub coefficients_comm: Vec<PolyComm<G>>,
```

Comment:
Instead of doing this, you should be able to add
```
#[serde_as(as = "[_; W]")]
```
and it will just work.

Reply:
Mmm... I will try that, I hadn't thought about using an underscore there. Thanks!